### ISSUE TITLE
Feature request: Can we get an official Firefox release too?

### ISSUE THREAD

**Issue by eddex** [Berry-Pool/nami-wallet#28]
Personally I prefer Firefox over Chrome based browsers. Could you please consider publishing a Firefox release too?

I am willing to help in case of any questions. I have my own Browser plugin that supports both browsers.

**Comment by alessandrokonrad** [comment]
You are free to test the extension in firefox. You can load it in manually. The problem is simply that it is slow and laggy, especially at the main wallet screen. I don't really know the reason unfortunately.

**Comment by eddex** [comment]
I tried to debug the extension and I was able to load it into Firefox. However, when I want to create a wallet I always run into an error. Not sure if I did something wrong in the setup tho.. Here's what I did:

- open `provider.js` and chnage it to the following:
```js
...
base: (node = NODE.mainnet) => node,
    key: (network = 'mainnet') => ({
      project_id: 'my-blockfrost-TESTNET-key'
    }),
...
```
Is this sufficient or do I need to change anything else to run Nami on the testnet?

- I proceeded to with the following to build the extension:
```sh
npm i
npm start
```

- Then I loaded the extension in debug mode, opened it and tried to restore a wallet. When restoring the wallet I get the following error:
```
Uncaught (in promise) ChunkLoadError: Loading chunk vendors-node_modules_emurgo_cardano-serialization-lib-browser_cardano_serialization_lib_js failed.
```
